documentation/modules/ref-address-space-example-exposing-endpoints.adoc

```diff
@@ -38,16 +37,16 @@ spec:
     - 10.0.0.0/8
 ----
 <1> (Required) The name of the endpoint. The name specified affects the name of the {KubePlatform} service to be created as well as the name of the endpoint in the status section of the `AddressSpace`.
-<2> (Required) The service names that must be configured for the `AddressSpace`.  The service can either be `messaging`, `console`, or `mqtt`. However, the `mqtt` service is supported for the `standard` address space type only.
-<3> (Required) The type of endpoint being exposed. The `loadbalancer` type will create an {Kubernetes} `LoadBalancer` service. Valid values are `route` and `loadbalancer`.
-<4> (Required) A list of the ports to be exposed on the `LoadBalancer` service. For `messaging`, `amqp` and `amqps` can be specified.
+<2> (Required) The service names that must be configured for the `AddressSpace`.  Valid values are `messaging`, `console`, and `mqtt`. However, the `mqtt` service is supported for the `standard` address space type only.
+<3> (Required) The type of endpoint being exposed. The `loadbalancer` type creates an {KubePlatform} `LoadBalancer` service. Valid values are `route` and `loadbalancer`.
+<4> (Required) A list of the ports to be exposed on the `LoadBalancer` service. For `messaging`, `amqp` and `amqps` are valid values.
 <5> (Optional) A set of key-value annotation pairs that are added to the `LoadBalancer` `Service` object.
 <6> (Optional) The allowed source ranges that are accepted by the load balancer.
-endif::[]
 
-== {KubePlatform} `LoadBalancer` service example
+ifeval::["{cmdcli}" =="oc"]
+== {KubePlatform} route example
 
-To expose `AddressSpace` endpoints as OpenShift routes, the `route` type is used:
+To expose `AddressSpace` endpoints as {KubePlatform} routes, the `route` type is used:
```

Comment:
This change is redundant as there is no such thing as a Kubernetes route - only OpenShift

Reply:
@rgodfrey I try to use variables for any product names, since that makes changes easy if a name changes (I realize it's arguable whether "OpenShift" would ever change, but was trying to be consistent in that practice throughout the docs)